content/java/concepts/math-methods/terms/floor/floor.md

```diff
@@ -0,0 +1,45 @@
+---
+Title: 'Floor'
+Description: 'Returns the largest integer value that is less than or equal to the argument. When the provided value is either an integer, zero, not a number, or positive or negative infinity.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags: 
+  - 'Documentation'
+CatalogContent: 
+  - 'learn-java/'
+  - 'paths/computer-science'
+---
+
+The **`Math.floor()`** method included in the `Math` class returns the largest integer value that is less than or equal to the argument.
+
+## Syntax
+
+
+```pseudo
+    Math.floor(x); // or passing values of variables
+```
+
+The return value is the same as `num` under the following conditions:
+
+- `num` is already an integer.
+- `num` is `NaN`, infinity, or zero (positive or negative).
+
+## Example 1
+
+In the case of double values, `Math.floor()` returns the next integer value below the provided double value. In the case of integers provided, these will return the same value as provided:
+
+```pseudo
```

Comment:
```suggestion
```java
```

Reply:
@Dusch4593 in the first review you asked for these to be pseudo:
> Let's use ```pseudo code blocks for the "Syntax" section. 😄
Just want to double check before reversing the previous change.

Reply:
@J0hnRJr This code block is not in the "Syntax" section; it is in the "Example 1" section and it should feature a \```java code block.

\```pseudo code blocks are strictly meant for describing the syntax of something as opposed to showcasing example of that something being applied.